internal/service/resource_handler.go

```diff
@@ -397,11 +431,28 @@ func (r *ResourceHandler) mapNodeItem(ctx context.Context,
 		return
 	}
 
+	labels, err := data.GetString(from, "label")
+	if err != nil {
+		return
+	}
+	labelsMap := data.GetLabelsMap(labels)
+
+	r.logger.Error(fmt.Sprintf("r.extensions: %v", r.extensions))
```

Comment:
You will probably want to remove this, or replace it with `r.logger.Debug(...)`.

Reply:
Good catch:) Removed.